### PR TITLE
Added Xterm and Xwayland. The X server now works.

### PR DESCRIPTION
**Pull request by JL2210**
See the description. There's really no better way to put it.

### CODE REVIEW

File: packages/xwayland.rb [hunk 1]
```diff
@@ -39,5 +40,6 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "ln", "-s", "#{CREW_PREFIX}/bin/Xwayland", "#{CREW_DEST_PREFIX}/bin/X"
```
Comment (uberhacker) [inline packages/xwayland.rb]: This looks good.  Can we bump the version?
```
- version '1.19.3'
+ version '1.19.3-1'
```

Comment (JL2210) [reply packages/xwayland.rb]: Done.

File: packages/xterm.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Xterm < Package
+  description 'The xterm program is a terminal emulator for the X Window System.'
+  homepage 'https://invisible-island.net/xterm/'
+  version '333'
+  source_url 'ftp://ftp.invisible-island.net/xterm/xterm-333.tgz'
```
Comment (uberhacker) [inline packages/xterm.rb]: `source_url 'https://invisible-mirror.net/archives/xterm/xterm-333.tgz'` ?

Comment (uberhacker) [reply packages/xterm.rb]: ftp should die a slow and painful death.

Comment (JL2210) [reply packages/xterm.rb]: Along with versioned packages?

Comment (uberhacker) [reply packages/xterm.rb]: What do you mean?  This is a new package, right?

Comment (JL2210) [reply packages/xterm.rb]: OK. Got it.

Comment (uberhacker) [reply packages/xterm.rb]: Curious to know, what is the time difference between:
`wget https://invisible-mirror.net/archives/xterm/xterm-333.tgz`
versus
`wget ftp://ftp.invisible-island.net/xterm/xterm-333.tgz` ?

Comment (JL2210) [reply packages/xterm.rb]: Done.

**Comment by JL2210** [comment]
The time difference should be less than an hour. If you get a versioned
tarball and not the latest code, you should be fine.

**Comment by uberhacker** [comment]
@JL2210: Xwayland has been upgraded already so we should be good.

**Pull request merged**